tmp/build/download-charts.sh

```diff
@@ -37,13 +37,11 @@ function retrieveIstioRelease() {
   fi
   mkdir -p "${RELEASE_DIR}"
 
-  if [ ! -f "${RELEASES_DIR}/${ISTIO_FILE}" ] ; then
-    (
-      echo "downloading Istio Release: ${ISTIO_URL}"
-      cd "${RELEASES_DIR}"
-      curl -LO "${ISTIO_URL}"
-    )
-  fi
+  (
+    echo "downloading Istio Release: ${ISTIO_URL}"
+    cd "${RELEASES_DIR}"
+    curl -LO "${ISTIO_URL}"
+  )
```

Comment:
It looks like this change snuck into the PR, it's not related to the task at hand

Reply:
heh, I had this change locally and got caught by git commit -a :/
Fixed, thanks.